docs/reference/09-qa-process-reference.md

```diff
@@ -2,6 +2,12 @@
 
 ## Generalities
 
+```{note}
+We always use the latest Go version available. Information about specific Go
```

Comment:
Clever way out of the maintenance burden! ;)

Reply:
There's probably a more clever way of solving this, but this is OK for now.